conf/junit.properties

```
@@ -0,0 +1,10 @@
+email.username=
+email.password=
+cf.email.recipients=
+cf.db.url=
+cf.db.username=
+cf.db.password=
+cf.filestore.directory=
+cf.filestore.advanced.812.44ac1c1a-dd0c-462f-b717-53324c0dd6f9=
+cf.filestore.advanced.1509.44ac1c1a-dd0c-462f-b717-53324c0dd6f9=
+cf.filestore.advanced.812.5a7b2083-2671-414b-8e5b-c4cd9dfa1f30=
```

newline at end of file

```suggestion
cf.filestore.advanced.812.5a7b2083-2671-414b-8e5b-c4cd9dfa1f30=

```